app/models/transaction.rb

```diff
@@ -0,0 +1,60 @@
+class Transaction < ApplicationRecord
+  #
+  ## Associations
+  #
+  belongs_to :account
+
+  #
+  ## Validations
+  #
+  validates(
+    :description,
+    presence: true,
+    length: {
+      maximum: 900
+    }
+  )
+  validates(
+    :amount,
+    presence: true
+  )
+  validates(
+    :transaction_type,
```

Comment:
@babarajay , Do you think `transaction_type` would be free text? How banks handle this? What if user type 'neft'? Will the comparison work?

Reply:
Do you think transaction_type would be free text? How banks handle this? What if user type 'neft'? Will the comparison work?
-> I've added one global variable for it, there will be no chance of free text or 'neft'; if somehow user added the free text; I'll add extra validation for it.